pkg/config/config.go

```diff
@@ -48,6 +48,7 @@ type Config struct {
 	RefreshToken string   `json:"refresh_token,omitempty" doc:"Offline or refresh token."`
 	Scopes       []string `json:"scopes,omitempty" doc:"OpenID scope. If this option is used it will replace completely the default scopes. Can be repeated multiple times to specify multiple scopes."`
 	TokenURL     string   `json:"token_url,omitempty" doc:"OpenID token URL."`
+	RhRegion     string   `json:"rh_region,omitempty" doc:"OCM region name."`
```

Comment:
i dont think we need to store this in the config since its implicit in the url. `ocm login --rh-region=singapore` results in the "singapore" region ocm instance url being stored in URL config variable. We want to try to limit region awareness to just the login command for now until we find a good need to explicitly store the selected region for use in other parts of the cli.

Reply:
Removed the JSON representation. Should we omit this?